frontend/android/src/main/res/layout/item.xml

```diff
@@ -6,27 +6,27 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:background="?android:attr/selectableItemBackground">
-    <TextView
+    <android.support.v7.widget.AppCompatTextView
         android:id="@+id/package_name"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
         android:layout_marginEnd="8dp"
         android:layout_marginStart="16dp"
         android:layout_marginTop="12dp"
-        android:autoSizeTextType="uniform"
+        app:autoSizeTextType="uniform"
```

Comment:
This should be sorted. This project uses https://github.com/square/java-code-styles which has the correct formatting rules for XML.